hasher-matcher-actioner/hmalib/indexers/lcc.py

```diff
@@ -33,22 +34,27 @@ def get_recent_index(cls, storage_path, signal_type) -> PDQIndex:
     @classmethod
     def build_index_from_last_24h(cls, signal_type, storage_path, bucket_width) -> None:
         """Create an index"""
-        d = timedelta(days=1)
+        with metrics.timer(metrics.names.lcc.build_index):
```

Comment:
This step is not building the index. It is merely fetching the data.

I'd say there are three stages:
- get_data (where we are getting data from timebucketizer)
- in_memory_processing (process the data L53-55
- build_index